tests/integration/explain.go

```diff
@@ -57,6 +57,7 @@ var (
 		"valuesNode":    {},
 		"viewNode":      {},
 		"lensNode":      {},
+		"operationNode": {},
```

Comment:
praise: Thanks for not forgetting this :P (easy to miss if the `operationNode` was not explainable node).